tests_pytest/keras_tests/e2e_tests/test_mct_bypass.py

```diff
@@ -12,3 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
+from model_compression_toolkit.gptq import keras_gradient_post_training_quantization
+from model_compression_toolkit.ptq import keras_post_training_quantization
+from tests_pytest.keras_tests._keras_base_tests.base_keras_mct_bypass import BaseKerasMCTBypassTest
+
+
+
+class TestKerasPTQMCTBypass(BaseKerasMCTBypassTest):
```

Comment:
Why implement something like this and not just add the API function as a parameter fixture?